types.json

```diff
@@ -1,4 +1,5 @@
 {
+    "AccountDataOf": "AccountData",
```

Comment:
Let's not forget to communicate with other teams that we yet again have changed the structure of the types file.

Reply:
Very good point!